appservice/src/slotFunctionOperations.ts

```diff
@@ -1,90 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-import type { WebSiteManagementModels } from '@azure/arm-appservice';
-import type { HttpOperationResponse, ServiceClient } from '@azure/ms-rest-js';
-
-/**
- * Temporary workaround because this isn't in azure sdk yet
- */
-export async function listFunctionsSlot(genericClient: ServiceClient, id: string): Promise<WebSiteManagementModels.FunctionEnvelopeCollection> {
```

Comment:
I think all of these APIs exist in the SDKs now, so we can remove this file. The names are different, e.g. `listInstanceFunctionsSlot`. I think all the calls are in `src/SiteClient.ts`.